<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPWD</class>
 <widget class="QDialog" name="AddPWD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add an entry</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Please select the password type you want to add :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="entryTypeCombo">
     <item>
      <property name="text">
       <string>Application Entry</string>
      </property>
      <property name="icon">
       <iconset resource="ressources.qrc">
        <normaloff>:/images/icons/app_entry.png</normaloff>:/images/icons/app_entry.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Network Entry</string>
      </property>
      <property name="icon">
       <iconset resource="ressources.qrc">
        <normaloff>:/images/icons/net_entry.png</normaloff>:/images/icons/net_entry.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Item Entry</string>
      </property>
      <property name="icon">
       <iconset resource="ressources.qrc">
        <normaloff>:/images/icons/item_entry.png</normaloff>:/images/icons/item_entry.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="hostLabel">
       <property name="text">
        <string>Host Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="hostEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="loginLabel">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="loginEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="portSpin">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="userLabel">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="userEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="appLabel">
       <property name="text">
        <string>Application Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="appEdit"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="itemEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="itemLabel">
       <property name="text">
        <string>Item Name</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="protocolLabel">
       <property name="text">
        <string>Protocol (optional)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="protocolEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddPWD</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddPWD</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
